happi/device.py

```diff
@@ -270,6 +270,9 @@ class Device(metaclass=InfoMeta):
                        enforce=str)
     parent = EntryInfo('If the device is a component of another, '
                        'enter the name', enforce=str)
+    lightpath = EntryInfo("If the device should be included in the ",
+                          "LCLS Lightpath", enforce=bool, default=False)
+    documentation = EntryInfo("Relevant documentation for the Device")
```

Comment:
What's the implication of not using `enforce` here?

Reply:
Yeah. It doesn't really leave the opportunity for multiple documentation sources, but you could argue that it should just point at a documentation homepage and child pages are linked from there.

I'll put it as a string if no one else here has opinions